### ISSUE TITLE
UnusedUse incorrectly reported

### ISSUE THREAD

**Issue by dhaarbrink** [slevomat/coding-standard#370]
Suppose you have the following class:

```php
<?php
declare(strict_types=1);

namespace App\JsonApi\Schema;

use App\Model\User;
use Neomerx\JsonApi\Schema\BaseSchema;

class UserSchema extends BaseSchema
{
    /**
     * @param User $user
     */
    public function getId($user): string
    {
        return (string)$user->id;
    }
}
```

Then I incorrectly get the following error:
> Type App\Model\User is not used in this file. (SlevomatCodingStandard.Namespaces.UnusedUses.UnusedUse)

The `User` is not used in code, that is correct. But it *is* used in an annotation, which is just as valid.

And in this case I cannot supply a typehint for the getId() method because it is defined in BaseSchema which is not under my control.

**Comment by ondrejmirtes** [comment]
Looks like you don't have `searchAnnotations` turned on: https://github.com/slevomat/coding-standard#slevomatcodingstandardnamespacesunuseduses-
